applications/crossbar/priv/couchdb/schemas/accounts.json

```diff
@@ -162,6 +162,26 @@
             "name": "Default Timezone",
             "required": false,
             "type": "string"
+        },
+        "vm_notify_callback_attempts": {
```

Comment:
@siplabs Howdy!  I would really prefer if these settings were nested and identical on all documents for a parameter controlling the same property(for example on accounts it is "vm_notify_callback_attempts" but on vmboxes it is "notify_callback_attempts").

I would propose the following object for all documents:
{
  ...
  "callback": {
    "voicemail": {
      "attempts": 1
      "disabled": false
      "interval": 300
      "timeout": 26
    }
  }
  ...
}

This helps set up for easy future expansion.  More callback modules could be added to the callback object, feature extensions of the voicemail feature are in that object.  ect...  This has been a lesson from experience which is why there are counter examples in some documents ;)